### PR TITLE
[frontend] create Latest Notification section and Avatar component and UserInfo Component.

### LINKED ISSUE #40: [frontend-latest-notifications] Create latest notifications section.

### PR DESCRIPTION
**Pull request by MostafaBalousha123**
**Explanation of all components in the photo**

```diff
- red: LatestNotification section

+ green: NotificationItem

# Black: Avatar

! yellow: UserInfo
```
Screenshot 2023-02-23 081417

### CODE REVIEW

**Review by MohammedOmar123 — APPROVED** [review]

**Comment by MohammedOmar123** [comment]
جنوووون

File: client/components/LatestNotification/NotificationList.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  return (
+    <div className='flex flex-col space-y-4'>
+      {notifications.map((e:INotificationsItem)=>
+      <NotificationItem
+          username={e.username}
+          image={e.image}
+          status={e.status}
+          postId={e.postId}
+          type={e.type}
+          />
```
Comment (Bas-Shiekh) [inline client/components/LatestNotification/NotificationList.tsx]: you need to add { key } to this component

File: client/components/Avatar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import Image from 'next/image'
+import { IAvatar } from '@/interfaces/props/IAvatar'
+import classNames from 'classnames'
+
+const Avatar:FC<IAvatar> = ({image, alt ,className, width, height}) => {
+return (
+        <Image
+            src={image}
+            alt={alt}
+            className={classNames('object-cover rounded-full',className)}
```
Comment (Bas-Shiekh) [inline client/components/Avatar/index.tsx]: try to add this shadow for avatar { shadow-[0_2px_4px_0px_rgba(1,1,1,0.5)] }

**Review by Bas-Shiekh — CHANGES_REQUESTED** [review]
great work

**Pull request merged**